docs/blog/newsletter-december-2017.rst

```diff
@@ -0,0 +1,90 @@
+
+
+#########################################
+Write the Docs Newsletter – December 2017
+#########################################
+
+Hi everybody! Kelly O. here, with our last issue of 2017! It's been a gigantic year for the Write the Docs community. Just in the last month, our members have gotten up to some amazing stuff, including:
+* a very successful first Australian event, which you can watch videos from `here <https://www.youtube.com/channel/UCPhWNEFb53x6PjnpRIYf1yg/videos>`_ and read about `here <https://ffeathers.wordpress.com/2017/11/25/doc-sprint-at-write-the-docs-day-australia/>`_.
+* a lively and informative third API the Docs event in Amsterdam, recaps from which are available `here <https://pronovix.com/api-docs-amsterdam-2017>`_.
+* and two new podcast episodes, which you can have a listen to `here <http://bit.ly/wtdpodcastepisode11>`_ and `here <http://podcast.writethedocs.org/2017/12/13/founding-principles-of-write-the-docs/>`_. That second one features WTD organizers Eric Holscher and Mikey Ariel as guests!
+
+Not only are we ending the year on a high note, but we also have a TON of great stuff coming up in 2018 as well!
+* We're already in full planning mode for the Portland and Prague conferences. The `call for proposals for Portland <http://www.writethedocs.org/conf/portland/2018/cfp/>`_ is open until January 10th, so if you haven't started your talk proposal, now's the time!
+* We've also started to line up sponsors for both conferences, so if your company is interested in showing off their love for good docs, have them drop us a line at `sponsorship@writethedocs.org <mailto:sponsorship@writethedocs.org>`_. You can read about our sponsorship packages in our `Portland <http://www.writethedocs.org/conf/portland/2018/sponsors/prospectus/>`_ and `Prague <http://www.writethedocs.org/conf/prague/2018/sponsors/prospectus/>`_ prospectuses (prospecti?).
+* And finally, early in 2018 we'll be launching a brand new Write the Docs Job Board! Our goal is to make it easy for documentarians to find their dream gigs, and employers to find their dream documentarians. If you'd be interested in helping us beta test the board in January – either as an employer or an applicant – we'd love to hear from you at `jobs@writethedocs.org <mailto:jobs@writethedocs.org>`_.
+
+Whew! Hopefully that gives you some things to look back on or forward to with excitement!
+
+Please note that the newsletter will be taking January off, to give our editorial crew a little break. We'll be back in the saddle with the February issue. In the meantime, have a happy holiday season and enjoy one last round of 2017 documentation stories, below!
+
+****************************************
+Help your contributors help your project
+****************************************
+
+Contribution guidelines are an important part of any open source project. A discussion about writing good contribution guidelines came up on Slack this month, and yielded some useful pearls of wisdom:
```

Comment:
useful is kinda redundant? I don't feel strongly about this

Reply:
👍